db/schema.rb

```diff
@@ -211,13 +211,14 @@
   end
 
   create_table "mentions", force: :cascade do |t|
-    t.integer "post_id",   limit: 4, null: false
-    t.integer "person_id", limit: 4, null: false
+    t.integer "mentions_container_id",   limit: 4,                    null: false
+    t.integer "person_id",               limit: 4,                    null: false
+    t.string  "mentions_container_type", limit: 255, default: "Post", null: false
   end
 
-  add_index "mentions", ["person_id", "post_id"], name: "index_mentions_on_person_id_and_post_id", unique: true, using: :btree
+  add_index "mentions", ["mentions_container_id"], name: "index_mentions_on_mentions_container_id", using: :btree
```

Comment:
you should also add `mentions_container_type` to this index, because it is never used without the type.